bika/lims/browser/analysisrequest/add2.py

```diff
@@ -320,6 +324,15 @@ def get_client(self):
             return context.getClient()
         return None
 
+    def get_sample(self):
+        """
+        Returns the Sample
+        """
```

Comment:
I know, it might sound pedantic, but stick to the same docstring style as the other methods, e.g.
```python
def get_sample(self):
    """Returns the Sample
    """
```